package/firmware/ipq-wifi/Makefile

```diff
@@ -50,7 +50,8 @@ ALLWIFIBOARDS:= \
 	netgear_wac510 \
 	plasmacloud_pa1200 \
 	plasmacloud_pa2200 \
-	qxwlan_e2600ac
+	qxwlan_e2600ac \
+	tp-link_deco_m5_eu_v3.0
```

Comment:
Typically, TP-Link uses the same hardware for all y in vx.y.

So, would v3 be enough here instead of v3.0?

Do you know whether non-eu versions are different? eu should only be added if it's different from the other v3, and when this is the case it should be last, i.e. Vendor Model Vx (EU) and vendor_model-vx-eu ...

Reply:
I did not submit the PR but I will reply since it may help with this effort.

Yes, v3 would do the trick. V3.2 and V3 have identical hardware. I believe V1 also, since TP-Link reuses the same firmware for V1, V3 and V3.2 variants. Those have NOR flash, while the V2 model has NAND flash. (Not sure if there are other differences)

International variants only differ in the provided power supply. The CA and US devices have a 120V adapter along with a barrel connector, the EU variant I believe has a USB-C port for power (and only power, not hooked up to USB logic).

Reply:
v1 and v3 should be separate then, and this should probably be named v3 without region ...